package.json

```diff
@@ -14,14 +14,19 @@
   },
   "dependencies": {
     "@reduxjs/toolkit": "^1.4.0",
+    "@rsksmart/ethr-did": "^1.1.1-beta.1",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.3.1",
```

Comment:
I found this tiny thing now. It should be dev dep. Should we remove 16 adapter?

Reply:
yes, good catch. I'll move it to dev and remove 16.